include/dali/core/dynlink_cuda.h

```diff
@@ -1886,9 +1886,9 @@ extern tcuMemcpy                       *cuMemcpy;
 extern tcuMemcpyPeer                   *cuMemcpyPeer;
 #endif
 
+extern tcuCtxCreate                    *cuCtxCreate;
```

Comment:
Please move it to where other Ctx functions are.

Reply:
Done